model.py

```diff
@@ -14,9 +17,9 @@ class User(db.Model):
 
     __tablename__ = "users"
 
-    user_id = db.Column(db.Integer, autoincrement=True, primary_key=True)
-    username = db.Column(db.String(50), nullable=False, unique=True)
-    password = db.Column(db.String(50), nullable=False)
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String, nullable=False, unique=True)
```

Comment:
If you have string size limits, there should be a good reason. Since usernames must be unique for your app, a limit like this is arbitrary. Same for passwords.

*Especially* for passwords.